lmdeploy/pytorch/kernels/pagedattention.py

```diff
@@ -627,6 +628,8 @@ def _kernel_meta():
     kernel_meta = _kernel_meta()
     is_decoding = q.shape[-3] == q_seqlens.size(0)
     if not is_decoding:
+        num_warps = 4
```

Comment:
https://triton-lang.org/main/getting-started/tutorials/06-fused-attention.html#sphx-glr-getting-started-tutorials-06-fused-attention-py

official kernel use `num_warps = 4` as default. And since this is only used in prefill, it should be acceptable even when it is suboptimal.